ansible-inventory-origin-39-aio

```diff
@@ -0,0 +1,32 @@
+
+[OSEv3:children]
+nodes
+masters
+etcd
+
+[OSEv3:vars]
+ansible_connection=local
+openshift_release=v3.9
+openshift_logging_install_logging=true
+short_version=3.9
+openshift_deployment_type=origin
+openshift_master_identity_providers=[{'mappingMethod': 'lookup', 'challenge': 'true', 'login': 'true', 'kind': 'AllowAllPasswordIdentityProvider', 'name': 'allow_all'}]
+# bug - doesn't use correct tag
+openshift_logging_image_version=v3.9
+openshift_logging_elasticsearch_proxy_image_version=v1.0.0
+
```

Comment:
I see you removed the following flags:
-openshift_hosted_logging_deploy=true
-openshift_image_tag=latest
-oreg_url=openshift/origin-${component}:latest
-openshift_logging_namespace=logging
-openshift_logging_es_cluster_size=1
-openshift_logging_image_prefix=docker.io/openshift/origin-
-deployment_type=origin
-openshift_logging_elasticsearch_proxy_image_prefix=docker.io/openshift/
-openshift_logging_elasticsearch_proxy_image_version=v1.1.0

Are they not needed anymore?

Reply:
see latest